patches/runtime/0012-Update-SDK-and-remove-RID-calculation-in-favor-of-Ru.patch

```diff
@@ -0,0 +1,418 @@
+From 8173f8d6070e3de59864e854f9d30b156b1bbfae Mon Sep 17 00:00:00 2001
+From: Viktor Hofer <viktor.hofer@microsoft.com>
+Date: Mon, 4 May 2020 11:45:15 +0200
+Subject: [PATCH 12/15] Update SDK and remove RID calculation in favor of
+ RuntimeInformation (#35538)
```

Comment:
I'm a little confused, I assume this a cherry-pick from a future release?

Reply:
Yes.  It's from https://github.com/dotnet/runtime/pull/35538.  I'll add that to the patch heading.